Assets/Scripts/HurtEffect.cs

```diff
@@ -5,95 +5,98 @@ public class HurtEffect : MonoBehaviour
 {
     private readonly float SHAKE_DEFORM_COEF = .2f;
```

Comment:
maybe use const in cases like this? https://stackoverflow.com/questions/755685/static-readonly-vs-const